### DISCUSSION TITLE
Ruleset translation for Wazuh

### DISCUSSION THREAD

**Discussion by n0rthl1ght** [SigmaHQ/pySigma-plugin-directory#20] [Announce Backends]
Hello everyone!
Can you add ruleset translation for Wazuh? Now I'm using another script to convert sigma rules to Wazuh xml ruleset and it's painful :(

**Comment by L1ghtn1ng** [comment]
This would be awesome to have if this can get looked at

**Comment by phantinuss** [comment]
@thomaspatzke: Can you migrate this over to pySigma? I seem to be missing permission to do so.

`@Thread`: This is the repo for Sigma rules. The [sigmac](https://github.com/SigmaHQ/legacy-sigmatools) converter will be end of life with the end of this year. Therefore don't expect anything be done for it. Someone would need to create a backend for [pySigma](https://github.com/SigmaHQ/pySigma) which is the successor of sigmac.

**Reply by thomaspatzke** [reply]
I move it to the plugin directory repository, where backend requests are collected.

**Comment by frack113** [comment]
I have works with OSSEC , I do not think you can create a backend to convert Sigma to the OSSEC/Wazuh.
Sigmac did not have one either.

**Reply by thomaspatzke** [reply]
Worked with Wazuh in the past and there's indeed a [rule language](https://documentation.wazuh.com/current/user-manual/ruleset/getting-started.html) that could be feeded by Sigma rules, at least by a subset.
